commune.lic

```diff
@@ -0,0 +1,412 @@
+	    'After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately',
+		'After a moment, you see a vision of your god who calls to you by name, "My child, though you may',
+		'After a moment, you see a vision of your god who calls to you by name, "Come here, my child',
+		'After a moment, you see a silent vision of your god',
+		'After a moment, you sense a slight pressure on your shoulder',
+		'After a moment, you see a vision of your god, though the visage is cloudy',
+		'After a moment, you sense that your god is pleased with your devotion',
+		'After a moment, you sense that your god knows your name',
+		'After a moment, you sense that your god is aware of your devotion',
+		'After a moment, you sense a distinct link between you and your god',
+		'After a moment, you sense that your efforts have not gone unnoticed',
+		'After a moment, you sense that your god is barely aware of you',
+		'You call out to your god, but there is no answer',
+		'You close your eyes and start to concentrate',
+		'You feel unclean and unworthy',
+		'You sense nothing special from your communing',
+		'You begin to act like you are trying to commune with your god']
+
+    arg_definitions = [
+      [
+       { name: 'Tamsine', regex: /^ta.*/i, variable: true, description: 'Increase TM vs. undead.' }
+      ],[
+       { name: 'Eluned', regex: /^e.*/i, variable: true, description: 'Create holy water.' }
+	  ],[
+	   { name: 'Kertigen', regex: /^k.*/i, variable: true, description: 'Bless weapon or person.' },
+	   { name: 'target', regex: /\w/i, optional: true, description: "Target weapon or person. Defaults to self." }
+	  ],[
+       { name: 'Hodierna', regex: /^h.*/i, variable: true, description: 'Staunch bleeding.' }
+	  ],[
+       { name: 'Meraud', regex: /^m.*/i, variable: true, description: 'Increase mana in room.' }
+	  ],[
+	   { name: 'Truffenyi', regex: /^tr.*/i, variable: true, description: 'Get or return favor orb for you. Get favor orb for others (kneeling). Proper offering required in right hand.' },
+	   { name: 'target', regex: /\w/i, optional: true, description: "Target person. Defaults to self. O" }
+	  ],
+	  [
+	   { name: 'Carve', regex: /^c.*/i, variable: true, description: 'Carve a prayer bead. Must have block or be able to forage a limb.' },
+	   { name: 'target', regex: /\w/i, optional: true, description: "Immortal for prayer bead. Defaults to Meraud. Must use full name." }
+	  ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    if args.to_h.empty?
+      display_args(arg_definitions)
+      return
+    end
+	
+	if args.Tamsine
+	  commune_tamsine
+	elsif args.Eluned
+	  commune_eluned
+	elsif args.Kertigen
+	  if args.target
+	    commune_kertigen("#{args.target}")
+      else
+	    commune_kertigen
+	  end
+	elsif args.Hodierna
+	  commune_hodierna
+	elsif args.Meraud
+	  commune_meraud
+	elsif args.Truffenyi
+	  if args.target
+	    commune_truffenyi("#{args.target}")
```

Comment:
There are two things here:
1. The only difference between `args.target` and `"#{args.target}"` is that the latter forces the value to be a string. However, args.target is already a string, so you don't need to do the string interpolation.
1. I'm not sure if you can change this to `commune_truffenyi(args.target)`. If that works, I think it would make the code clearer. If it doesn't, I recommend trying a different style that might also work:
```
def commune_truffenyi(target)
  target = target || @self
end
...
commune_truffenyi(args.target)
```